test/CMakeLists.txt

```diff
@@ -55,4 +55,6 @@ include_directories(
 )
 
 add_subdirectory(${SYCLBLAS_TEST}/unittest)
-#add_subdirectory(${SYCLBLAS_TEST}/exprtest)
+if(ENABLE_EXPRESSION_TESTS)
```

Comment:
Please use option and set the default value for ENABLE_EXPRESSION_TESTS

Reply:
Added to main CMakeLists